packages/react/src/hooks/useAuth.ts

```diff
@@ -132,6 +140,7 @@ export const useAuth: UseAuth = () => {
       orgSlug: null,
       signOut,
       getToken,
+      authStateKey: null,
```

Comment:
Shouldn't we use a hash here as well? It is still an "authState".

Reply:
🔧  Same.

Reply:
@desiprisg https://github.com/clerkinc/javascript/pull/1493#discussion_r1266511299